signer/src/testing/wsts.rs

```diff
@@ -517,7 +518,7 @@ impl SignerSet {
         &self,
         storage: &S,
         chain_tip: &model::BitcoinBlockHash,
-        aggregate_key: PublicKey,
+        shares: &EncryptedDkgShares,
```

Comment:
qq: is this just a matter of style, or we gain something by using shares instead of the aggregated key?

Reply:
Kinda. This is a change in anticipation of another change that will make things a little more clear. I think we should store more data in the database so that the different public keys can be used (I talk about the change in https://github.com/stacks-network/sbtc/issues/614).